strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/testing/storage/repository/TestRepositoryManagementApplicationContext.java

```diff
@@ -232,9 +226,11 @@ private void unlock()
             }
 
             ReentrantLock lock = entry.getValue();
-            lock.unlock();
-
-            logger.info(String.format("Test resource [%s] unlocked.", id));
+            if (lock.isLocked())
```

Comment:
@ptirador why this was needed?

Reply:
it seems that this is related with issue that @fuss86 reported recently within [PR](https://github.com/strongbox/strongbox/pull/1417). if so then I am working to fix it, and we probably need to wait and then rebase on fixed master.

Reply:
Yeah, I added this becase I was having lock issues when several neste groups where created by using annotations. In this case, there was an error while trying to unlock a repository group which was already unlocked.

Reply:
fix merged into master, so it should work after rebase